### ISSUE TITLE
GMScreen - unwanted margins make it float off the bottom of the screen

### ISSUE THREAD

**Issue by apoapostolov** [Rayuaz/twilight-ui#27]
Maybe an unwanted result of applying margins to all windows, but GMScreen should stay firmly connected to the bottom of the screen.

![image](https://user-images.githubusercontent.com/4083812/125994105-4c707d53-af95-4798-b309-9513cb74646e.png)

**Comment by apoapostolov** [comment]
```
.window-app .tab.active { margin-bottom: 0px !important; }
```

**Comment by apoapostolov** [comment]
Another GMScreen issue, padding applied in the cells makes them look bad and crooked:

This is with latest Twilight, no change on my side:

![image](https://user-images.githubusercontent.com/4083812/125995740-27f221ad-6ed9-4859-ad47-163bb64c8e70.png)

This is how it should be

![image](https://user-images.githubusercontent.com/4083812/125996272-2e3a32e0-9a8b-4f9a-9dd6-eaa934576bb9.png)

```
.gm-screen-app .gm-screen-grid-cell-content .editor { height: fit-content !important; }
```

**Comment by Rayuaz** [comment]
Thanks for the feedback, dude! It has been fixed in v1.04.
